ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -88,7 +88,8 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   // convert Perses dashboard format to be ECharts compatible
   const yAxis = {
     show: y_axis?.show ?? DEFAULT_Y_AXIS.show,
-    min: y_axis?.min, // leaves min and max undefined by default to let ECharts calcualate
+    // dataMin sets minimum axis label relative to data instead of zero
+    min: y_axis?.min === 'ScaleToData' ? 'dataMin' : y_axis?.min, // leaves min and max undefined by default to let ECharts calcualate
```

Comment:
We might want to add some padding (i.e. make the y-axis minimum lower than the data minimum) 

But I think this is fine for starting / might be fine in general

Reply:
Yeah good point! Updated the approach to account for this